diff --git a/server/Dockerfile b/server/Dockerfile
index 2f16346b296dc464933fa575b6c9f2997ee4948f..6912c0b71cc1d5dc0df1a809341b664b4a9c8048 100644
--- a/server/Dockerfile
+++ b/server/Dockerfile
@@ -1,6 +1,6 @@
-FROM registry.access.redhat.com/ubi8-minimal
+FROM richardjkendall/centos8arm

-ENV KEYCLOAK_VERSION 15.0.0
+ENV KEYCLOAK_VERSION 15.0.2
 ENV JDBC_POSTGRES_VERSION 42.2.5
 ENV JDBC_MYSQL_VERSION 8.0.22
 ENV JDBC_MARIADB_VERSION 2.5.4
